report.md

```diff
@@ -4,123 +4,157 @@
-Additionally enabling to export the resultant data into `CSV` format to
-be used outside R (_e.g._, with Excel). 
-A business readable data quality check dictionary is used in the pipeline to 
-clean the data. The non-validated data are
-either modified or removed according to the such criteria. Subsequently, an
-auto-generated data quality report is produced to report the key fields
-missingness and data sanity by each site and unit. By the end of the pipeline the
-researchers obtain a R table which is queryable and
-validated. We also have demonstrated the ability of deriving data products, such as
-SOFA score from physiology data and identification of sepsis. A publication 
-about this process is in due course.
-
-We believe a robust, understandable and accessible software is
-the first step of solid research. Therefore, beyond the functionalities, 
-we focus in the usability and sustainability of
-the software produced. All of our development is constantly updated in the
-GitHub platform which is accessible to all the allowed users.  The installation process of
-our software aims to simplicity. The code is self-explanatory and well
-documented empowering users and future
-developers. Testings are conducted after any modification of code automatically
-on the Travis CI system. Overall this ensures the software to be a gateway to the data which
-plays a critical role in the future success of the community.
+bridge this gap and deliver ready-to-use data and data
+manipulation tools for researchers. 
+
+We have built a pipeline which converts `XML` raw data to the queryable RData format.
+The resulting data can also be exported into a `CSV` format to
+allow it to be used outside R (_e.g._, with Excel).
+
+A ``data quality check dictionary'' is used in the pipeline to 
+clean the data. Invalid or corrupted data is
+either modified or removed according to these criteria, which are presented in a 
+``business readable'' file format allowing researchers to design and verify the criteria
+for record inclusion. An auto-generated data quality report is produced 
+to report on the rate of ``missingness'' of key fields, and on data sanity by site and unit. 
+
+By the end of the pipeline
+researchers obtain an R table which is queryable and
+validated. 
+
+We also have demonstrated the ability of programmatically
+deriving data products from this pipeline, 
+such as both automatic generation of a SOFA score
+and identification of sepsis. A publication 
+about this process is in preparation.
+
+We believe robust, understandable and accessible software is
+the first step of solid computationally-based research. 
+Therefore, in addition to the functionalities described above, 
+we have also focused on the usability and sustainability of
+the software produced. 
+
+All of our development is constantly updated in the
+GitHub platform which is accessible to all the collaboration.  
```

Comment:
`accessible to all the collaborators`? or `to the whole collaboration`? I am not so sure of any of the options...